plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/ConnectionProfileOraclePage.java

```diff
@@ -10,27 +10,24 @@
  ******************************************************************************/ 
 package org.jboss.reddeer.eclipse.datatools.ui.wizard;
 
-import org.jboss.reddeer.swt.impl.button.CheckBox;
-import org.jboss.reddeer.swt.impl.text.LabeledText;
-
 /**
  * A wizard page for creating new Oracle database profile.
  * 
  * 
- * @author apodhrad
+ * @author apodhrad, mlabuda@redhat.com
  * 
  */
 public class ConnectionProfileOraclePage extends ConnectionProfileDatabasePage {
```

Comment:
name & package should be eclipse.datatools.enablement.oracle.internal.ui.OracleDBProfileDetailsWizardPage